packages/orchestrator-ui-components/src/pages/workflow/WFOStartWorkflowPage.tsx

```diff
@@ -35,7 +35,7 @@ type StartWorkFlowPayload =
 
 interface WFOStartWorkflowPageProps {
     workflowName: string;
-    startWorkflowPayload: StartWorkFlowPayload;
+    startWorkflowPayload?: StartWorkFlowPayload | undefined;
```

Comment:
? and undefined is double the same, in case of component props i would go for "?"

Reply:
Fixed; again silly names here -> StartWorkFlowPayload:

- it starts a process
- All other place use Workflow instead of WorkFlow
- StartWorkFlowPayload can also be used for Modify/Terminate?